tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -379,9 +379,9 @@ func (a *AwsProvider) findWindowsWorkerSg(infraID string) (string, error) {
 	return *sgs.SecurityGroups[0].GroupId, nil
 }
 
-// getMyIp get the external IP of user's machine from https://checkip.amazonaws.com and returns an address or an error.
+// GetMyIp get the external IP of user's machine from https://checkip.amazonaws.com and returns an address or an error.
 // The 'checkip' service is maintained by Amazon.
-func getMyIp() (string, error) {
+func GetMyIp() (string, error) {
```

Comment:
Such changes should be called out in your commit message mentioning why you are making them.